server.js

```diff
@@ -68,7 +76,7 @@ const socketServer = new WebSocket.Server({ port: 3030 });
 console.log("listening on 3030");
 
 // FIXME: Comment the following line when you want to test with PCAP file
-// pktStreamer.emit("packet");
+// pktStreamer.emit("packet"); 
 
 socketServer.on("connection", (socketClient) => {
```

Comment:
Also below line number - This should be wrapped inside an if loop if `--live` is false.